src/inpage-content.js

```diff
@@ -0,0 +1,120 @@
+
+const url = window.location.href;
+// const action = url.split('?')[1].split('=')[1];
+
+const action = parseQuery(url).action;
+const btnLink = parseQuery(url).btnlink;
+
+const loginItem = document.getElementById("fbc-login");
+const emailItem = document.getElementById("fbc-email");
+
+function parseQuery(queryString) {
+    var query = {};
+
+    var pairs = queryString.split('?')[1].split('&');
+    for (var i = 0; i < pairs.length; i++) {
+        var splt = pairs[i].split('=');
+        query[splt[0]] = splt[1]
+    }
+    return query;
+}
+
+
+if (action === "login") {
+    loginItem.classList.remove("is-hidden");
+}
+
+if (action === "email") {
+    emailItem.classList.remove("is-hidden");
+}
+
+// Header String
+const fbcTitle = document.querySelector(".fbc-title");
+fbcTitle.innerHTML = browser.i18n.getMessage("facebookContainer");
```

Comment:
Issue (blocking): Use `textContent` instead of `innerHTML` across all instances: 

[It gets better performance and can prevent XSS attacks.](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innerhtml)